modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeSourceProcessorTest.java

```diff
@@ -64,13 +64,15 @@ public void testUpgradeJavaCheck() throws Exception {
 	}
 
 	@Test
-	public void testUpgradeJavaExtractTextMethodCheck() throws Exception {
-		test("upgrade/UpgradeJavaExtractTextMethodCheck.testjava");
+	public void testUpgradeJavaServiceTrackerListCheck() throws Exception {
+		test("upgrade/UpgradeJavaServiceTrackerListCheck.testjava");
```

Comment:
@TamyrisTorres, is this change really related to this pull?

In addition, this part of the commit should actually be part of the second commit of this pull, right?